### PR TITLE
Update the documentation about NLPModelsJuMP

### PR DESCRIPTION
**Pull request by amontoison**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -83,6 +83,6 @@ jobs:
       after_success: skip
 
 after_success:
-  - julia -e 'if Sys.islinux() && string(VERSION)[1:3] == "1.4"
```
Comment (amontoison) [inline .travis.yml:86]: Abel, do you have an idea for avoiding this update at each new release of Julia ?

Comment (dpo) [reply .travis.yml:86]: Use just 1 ?!

File: .travis.yml [hunk 2]
```diff
@@ -83,6 +83,6 @@ jobs:
...
+  - julia -e 'if Sys.islinux() && string(VERSION)[1:3] == "1.5"
```
Comment (abelsiqueira) [inline .travis.yml:86]:
```suggestion
  - julia -e 'if Sys.islinux()
```
There is no harm in building every linux, so we can just avoid deciding which release.

File: .travis.yml (hunk 1 above)
Comment (amontoison) [reply .travis.yml:86]: `1` is only used by Travis. `VERSION` is a variable defined by Julia and it is 1.0.5, 1.4.2, 1.5.0, etc...

**Comment by abelsiqueira** [comment]
Thanks

**Pull request merged**